README.md

```diff
@@ -81,6 +89,8 @@ Even if `options` is empty or undefined or `options.tokens` is `false`, it assig
 **Example :**
 
 ```js
+import espree from "espree";
+
```

Comment:
it should be `import * as espree from "espree"` - as espree do not have `export defautl`.

or just `import {tokenize} from "espree"`